### PR TITLE
Add  ChoudharyTara.json

### PR DESCRIPTION
**Pull request by ChoudharyTara**
<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/765"></a>

### CODE REVIEW

**Review by krishguptadev — APPROVED** [review]

File: public/data/ChoudharyTara.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ChoudhariTara?t=Jhx7vWfN30_xYZJaOKr0uw&s=09",
+      "icon": "Twitter"
+    },
+    {
+      "name": "Reach out me",
+      "url": "choudharytara0208@gmail.com",
+      "icon": "Envelope"
+    },
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:25]:
```suggestion
    }
```

File: public/data/ChoudharyTara.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "Conenct with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/tara-choudhary-257156219",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ChoudhariTara?t=Jhx7vWfN30_xYZJaOKr0uw&s=09",
+      "icon": "Twitter"
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:19]:
```suggestion
      "icon": "twitter"
```

File: public/data/ChoudharyTara.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name": "Reach out me",
+      "url": "choudharytara0208@gmail.com",
+      "icon": "Envelope"
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:24]:
```suggestion
      "icon": "envelope"
```

File: public/data/ChoudharyTara.json (hunk 1 above)
Comment (ChoudharyTara) [reply public/data/ChoudharyTara.json]: @krishguptadev Thanks!

File: public/data/ChoudharyTara.json (hunk 3 above)
Comment (ChoudharyTara) [reply public/data/ChoudharyTara.json]: Done 👍

File: public/data/ChoudharyTara.json (hunk 2 above)
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:19]:
```suggestion
      "icon": "twitter"
```

File: public/data/ChoudharyTara.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Conenct with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/tara-choudhary-257156219",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ChoudhariTara?t=Jhx7vWfN30_xYZJaOKr0uw&s=09",
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:18]:
```suggestion
      "url": "https://twitter.com/ChoudhariTara",
```

File: public/data/ChoudharyTara.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    },
+    {
+      "name": "Reach out me",
+      "url": "choudharytara0208@gmail.com",
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:23]:
```suggestion
      "url": "mailto:choudharytara0208@gmail.com",
```

**Review by krishguptadev — APPROVED** [review]

File: public/data/ChoudharyTara.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  "bio": "I'm Passionate and enthusiast about exploring new things and web development",
+  "avatar": "https://github.com/ChoudharyTara.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ChoudharyTara",
+      "icon": "github"
+    },
+    {
+      "name": "Conenct with me on LinkedIn",
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:12]:
```suggestion
      "name": "Connect with me on LinkedIn",
```

File: public/data/ChoudharyTara.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      "url": "https://www.linkedin.com/in/tara-choudhary-257156219",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ChoudhariTara",
+      "icon": "twitter"
+    },
+    {
+      "name": "Reach out me",
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:22]:
```suggestion
      "name": "Reach out to me via email",
```

File: public/data/ChoudharyTara.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Tara Choudhary",
+  "bio": "I'm Passionate and enthusiast about exploring new things and web development",
```
Comment (krishguptadev) [inline public/data/ChoudharyTara.json:3]: Should be lowercase, (Punctuations in English)

```suggestion
  "bio": "I'm passionate and enthusiast about exploring new things and web development",
```

Comment (ChoudharyTara) [reply public/data/ChoudharyTara.json]: Done!

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

**Review by Vyvy-vi — APPROVED** [review]
Looks Good To Me!

**Pull request merged**